### PR TITLE
fix(external-service): exclude endpoint if service type externalName is set

### PR DESCRIPTION
**Pull request by mateuszdrab**
This stops the endpoint from being created when using externalName

### CODE REVIEW

**Comment by philippwaller** [comment]
Hi @mateuszdrab

Thank you very much. In the meantime I have fixed the CI pipeline. Can you please rebase your changes and bump the chart & app version (Chart.yaml) to 1.0.4. I will merge the PR in the next few hours.

File: charts/external-service/templates/endpoint.yaml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+{{- if ne  (lower .Values.service.type) "externalname" }}
```
Comment (philippwaller) [inline charts/external-service/templates/endpoint.yaml:1]: Very good approach!

**Comment by mateuszdrab** [comment]
I messed this up so bad, I think I'll start again unless you can just squash those commits.

**Comment by philippwaller** [comment]
No worries :smile:. I will squash and merge the changes. The pipeline will run for some time. As soon as all tests are completed, I will finalize the PR. Thanks again for your contribution and bug fixes!

**Review by philippwaller — APPROVED** [review]

**Pull request merged**